src/runtime/entitlements-manager.js

```diff
@@ -400,6 +404,17 @@ export class EntitlementsManager {
     return this.createEntitlements_('', [], isReadyToPay);
   }
 
+  /**
+   * Persist swgUserToken in local storage if entitlements and swgUserToken exist
+   * @param {string} swgUserToken
```

Comment:
I think swgUserToken can also be undefined here.

Reply:
Added ?

Reply:
I think you want |undefined. '?' indicates it's nullable.